src/modules/application/components/Header/Header.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import useFelaEnhanced from "hooks/useFelaEnhanced";
+import { ReactComponent as Logo } from "assets/vectors/img_launch_placeholder.svg";
+
+import * as felaRules from "./Header.styles";
+import { FormattedMessage } from "react-intl";
+
+const Header = () => {
+  const { styles } = useFelaEnhanced(felaRules);
+
+  return (
+    <header className={styles.container}>
+      <Logo />
+      <div>
+        <FormattedMessage id="page.header.madeWithLove" />{" "}
+        <a href="https://ackee.cz" target="_blank" rel="noreferrer">
```

Comment:
Ackee web is a safe harbor so we can use `noopener` so they know the user came from here in analytics :)

Reply:
Ok, it was linted in IDE without the `noreferrer`, due to security risk. Changed it to `noopener` ✅

Reply:
The `noopener` should be fine because it works for most of the browsers the same as `noreferrer` in terms of the security risks :)